easybuild/easyconfigs/p/ProteinMPNN/ProteinMPNN-1.0.1_20230627-foss-2022a-CUDA-11.7.0.eb

```
@@ -0,0 +1,43 @@
+# Author: Lara Peeters (UGent)
+
+easyblock = 'Tarball'
+
+name = 'ProteinMPNN'
+version = '1.0.1_20230627'
+local_commit = '8907e6671bfbfc92303b5f79c4b5e6ce47cdef57'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://github.com/dauparas/ProteinMPNN'
+description = """A deep learning based protein sequence design method is 
+described that is widely applicable to current design challenges and shows 
+outstanding performance in both in silico and experimental tests.
+"""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+github_account = 'dauparas'
+source_urls = [GITHUB_SOURCE]
+sources = [{'download_filename': '%s.tar.gz' % local_commit, 'filename': SOURCE_TAR_GZ}]
+checksums = ['2e11f385074facb06e12496acde80756b34504f0f70bf78714fb07fe9244f398']
+
+dependencies = [
+    ('CUDA', '11.7.0', '', SYSTEM),
+    ('PyTorch', '1.12.0', '%(versionsuffix)s'),
```

@laraPPr sorry to be a pain, there's no need to use a template here, it can be just `versionsuffix`
```suggestion
    ('PyTorch', '1.12.0', versionsuffix),
```